modules/blocks/verify.js

```diff
@@ -221,7 +222,10 @@ __private.verifyAgainstLastNBlockIds = function(block, result) {
  * @returns {Array} result.errors - Array of validation errors
  */
 __private.verifyVersion = function(block, result) {
-	if (block.version > 0) {
+	if (
+		block.version !== blockVersion.currentBlockVersion &&
+		!blockVersion.inExceptions(block.version, block.height)
```

Comment:
Can we move this complete logic to `logic/block_version.isValid(height, version)` that will be more useful for future.